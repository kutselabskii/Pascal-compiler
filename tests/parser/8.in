program abc;

var a, b, c: integer;
    d, e: real;
    f: char;
    g: string;
begin
end

.