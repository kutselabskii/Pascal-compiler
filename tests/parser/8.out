|__ program
    |-- abc //Program_heading
    |__ //Block
        |-- //Declaration part
        |   |-- var
        |   |   |-- a
        |   |   |   |__  : integer
        |   |   |__ b
        |   |       |__  : integer
        |   |   |__ c
        |   |       |__  : integer
        |   |   |__ d
        |   |       |__  : real
        |   |   |__ e
        |   |       |__  : real
        |   |   |__ f
        |   |       |__  : char
        |   |   |__ g
        |   |       |__  : string
        |__ //Statement part
            |__ //Statement sequence
